discover/index.html

```diff
@@ -27,32 +27,32 @@ <h2 class="text-left"><i class="fas fa-fw fa-plane"></i> Getting Started</h2>
   <li>WMTS: <code>https://discover.agrc.utah.gov/login/path/{quad-word}/wmts/1.0.0/WMTSCapabilities.xml</code>
   <li>Tiles: <code>https://discover.agrc.utah.gov/login/path/{quad-word}/tiles/{service}/{z}/{x}/{y}</code>
 </ul>
-<p>AGRC has compiled resources to help you successfully use the Discover server. You can find in depth information about the following on the <a href="{{ "/discover/resources/" | prepend: site.baseurl }}">Discover resource</a> page.</p>
+<p>To learn more about using the Discover Server, be sure to visit our <a href="{{ "/discover/resources/" | prepend: site.baseurl }}">Discover Server Resources and Information</a> page. The following topics are covered in more depth on that page, but we've compiled them here as well for your convenience.</p>
 <ul class="dotless">
-  <li><a href="{{ "/discover/resources/#coordinate-system-and-datum" | prepend: site.baseurl }}">Coordinate System and Datum</a>
-  <li><a href="{{ "/discover/resources/#horizontal-positional-accuracy" | prepend: site.baseurl }}">Horizontal Positional Accuracy</a>
-  <li><a href="{{ "/discover/resources/#adding-a-wmts-or-wms-service-to-arcmap" | prepend: site.baseurl }}">Adding a service in an esri Product</a>
+  <li><a href="{{ "/discover/resources/#coordinate-system-and-datum" | prepend: site.baseurl }}">Coordinate system and datum</a>
+  <li><a href="{{ "/discover/resources/#horizontal-positional-accuracy" | prepend: site.baseurl }}">Horizontal positional accuracy</a>
+  <li><a href="{{ "/discover/resources/#adding-a-wmts-or-wms-service-to-arcmap" | prepend: site.baseurl }}">Adding a service in an Esri product</a>
   <li><a href="{{ "/discover/resources/#adding-a-wmts-or-wms-service-to-a-web-map" | prepend: site.baseurl }}">Adding a service in ArcGIS Online Web Maps</a>
   <li><a href="{{site.baseurl}}{% post_url 2015-12-21-using-agrcs-new-web-mercator-services-in-your-web-maps %}">Use in JavaScript/Web Mapping application</a>
   <li><a href="{{ "/discover/resources/#adding-a-wms-service-to-cad" | prepend: site.baseurl }}">Adding a service in CAD</a>
-  <li><a href="{{ "/discover/resources/#proarcmap-user-considerations" | prepend: site.baseurl }}">ArcGIS Pro Considerations</a>
+  <li><a href="{{ "/discover/resources/#proarcmap-user-considerations" | prepend: site.baseurl }}">ArcGIS Pro considerations</a>
   <li><a href="{{ "/discover/resources/#printing-web-maps-with-discover-services" | prepend: site.baseurl }}">Printing Discover services</a>
-  <li><a href="{{ "/discover/resources/#google-flight-dates" | prepend: site.baseurl }}">Flight Dates</a>
-  <li><a href="{{ "/discover/resources/#requests-for-on-premise-use" | prepend: site.baseurl }}">On Premise use</a>
+  <li><a href="{{ "/discover/resources/#google-flight-dates" | prepend: site.baseurl }}">Flight dates</a>
+  <li><a href="{{ "/discover/resources/#requests-for-on-premise-use" | prepend: site.baseurl }}">On-premises use</a>
```

Comment:
Here, I'm thinking we mean "premises," correct? I know it sounds a little weird with "on-premises" vs "on-premise," but "premise" and "premises" have different meanings.

Reply:
it should be the one that means you want to use the data in your own office

Reply:
Ok, "premises" then!